lib/hanami/commands/new/abstract.rb

```diff
@@ -93,6 +93,10 @@ def git_dir_present?
           File.directory?(target.join('.git'))
         end
 
+        def git_available?
+          system('which', 'git', out: File::NULL)
```

Comment:
@AlfonsoUceda You are right. Maybe a first step could be to have `git_available?` always return `true` on windows for the moment, and only do the check on *nix.